app/src/main/java/org/oppia/app/help/faq/FAQActivity.kt

```diff
@@ -16,7 +17,12 @@ class FAQActivity : InjectableAppCompatActivity() {
     super.onCreate(savedInstanceState)
     activityComponent.inject(this)
     faqActivityPresenter.handleOnCreate()
-    title = getString(R.string.frequently_asked_questions)
+    title = getString(R.string.FAQs)
```

Comment:
Do not set title here. I know you have used `<include>` in xml and because of which you are using this code. But instead I would suggest you to checkout the of `exploration_activity` and create toolbar in that way. Also, we should include data binding in this file at this stage only.

Reply:
done